nyx/chain.json

```diff
@@ -45,6 +45,10 @@
       }
     ]
   },
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nyx/images/nyx.png",
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nyx/images/nyx.svg"
```

Comment:
same with this svg, looks like a bad recreation attempt and should probably be omitted